client/me/purchases/manage-purchase/plan-details/index.jsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * External Dependencies
+ */
+import { connect } from 'react-redux';
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import Card from 'components/card';
+import ClipboardButtonInput from 'components/clipboard-button-input';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import QueryPluginKeys from 'components/data/query-plugin-keys';
+import { getPurchase, isDataLoading } from '../../utils';
+import { isJetpackPlan, isFreeJetpackPlan } from 'lib/products-values';
+import { getPluginsForSite } from 'state/plugins/premium/selectors';
+
+class PurchasePlanDetails extends Component {
+	renderPluginLabel( slug ) {
+		switch ( slug ) {
+			case 'vaultpress':
+				return this.props.translate( 'Backups and security scanning API key' );
+			case 'akismet':
+				return this.props.translate( 'Anti-spam API key' );
+		}
+	}
+
+	render() {
+		if ( isDataLoading( this.props ) || ! this.props.selectedSite ) {
```

Comment:
Would it make sense to display a placeholder instead of nothing, for example in a situation where purchases and site have loaded, but plugin keys haven't?